providers/aws/services/vpc/vpc_service_test.py

```diff
@@ -0,0 +1,254 @@
+        audit_info = self.set_mocked_audit_info()
+        vpc = VPC(audit_info)
+        assert (
+            len(vpc.vpcs) == 24
+        )  # Number of AWS regions + created VPC, one default VPC per region
+
+    # Test VPC Describe Flow Logs
+    @mock_ec2
+    def test__describe_flow_logs__(self):
+        # Generate VPC Client
+        ec2_client = client("ec2", region_name=AWS_REGION)
+        new_vpc = ec2_client.create_vpc(CidrBlock="10.0.0.0/16")["Vpc"]
+        # Create VPC Flow log
+        ec2_client.create_flow_logs(
+            ResourceType="VPC",
+            ResourceIds=[new_vpc["VpcId"]],
+            TrafficType="ALL",
+            LogDestinationType="cloud-watch-logs",
+            LogGroupName="test_logs",
+            DeliverLogsPermissionArn="arn:aws:iam::"
+            + str(AWS_ACCOUNT_NUMBER)
+            + ":role/test-role",
+        )
+        # VPC client for this test class
+        audit_info = self.set_mocked_audit_info()
+        vpc = VPC(audit_info)
+        # Search created VPC among default ones
+        for vpc in vpc.vpcs:
+            if vpc.id == new_vpc["VpcId"]:
+                assert vpc.flow_log == True
+
+    # Test VPC Describe VPC Peering connections
+    @mock_ec2
+    def test__describe_vpc_peering_connections__(self):
+        # Generate VPC Client
+        ec2_client = client("ec2", region_name=AWS_REGION)
+        # Create VPCs peers
+        vpc = ec2_client.create_vpc(CidrBlock="10.0.0.0/16")
+        peer_vpc = ec2_client.create_vpc(CidrBlock="11.0.0.0/16")
+        vpc_pcx = ec2_client.create_vpc_peering_connection(
+            VpcId=vpc["Vpc"]["VpcId"], PeerVpcId=peer_vpc["Vpc"]["VpcId"]
+        )
+        vpc_pcx_id = vpc_pcx["VpcPeeringConnection"]["VpcPeeringConnectionId"]
+
+        vpc_pcx = ec2_client.accept_vpc_peering_connection(
+            VpcPeeringConnectionId=vpc_pcx_id
+        )
+        # VPC client for this test class
+        audit_info = self.set_mocked_audit_info()
+        vpc = VPC(audit_info)
+        assert len(vpc.vpc_peering_connections) == 1
+
+    # Test VPC Describe VPC Peering connections
+    @mock_ec2
+    def test__describe_vpc_peering_connections__(self):
+        # Generate VPC Client
+        ec2_client = client("ec2", region_name=AWS_REGION)
+        # Create VPCs peers
+        vpc = ec2_client.create_vpc(CidrBlock="10.0.0.0/16")
+        peer_vpc = ec2_client.create_vpc(CidrBlock="11.0.0.0/16")
+        vpc_pcx = ec2_client.create_vpc_peering_connection(
+            VpcId=vpc["Vpc"]["VpcId"], PeerVpcId=peer_vpc["Vpc"]["VpcId"]
+        )
+        vpc_pcx_id = vpc_pcx["VpcPeeringConnection"]["VpcPeeringConnectionId"]
+
+        vpc_pcx = ec2_client.accept_vpc_peering_connection(
+            VpcPeeringConnectionId=vpc_pcx_id
+        )
+        # VPC client for this test class
+        audit_info = self.set_mocked_audit_info()
+        vpc = VPC(audit_info)
+        assert len(vpc.vpc_peering_connections) == 1
```

Comment:
Also check that the values created are the same that the received in the client

Reply:
Done